src/pages/whats-happening/changelog-and-roadmap/index.mdx

```diff
@@ -33,25 +44,129 @@ Carbon’s future.
 </Column>
 </Row>
 
-## Carbon team
+## Releases
 
-<Row className="resource-card-group">
-<Column colMd={4} colLg={4} noGutterSm>
-  <ResourceCard
-    subTitle="Team 2020 Q4 OKRs"
-    href="https://app.zenhub.com/workspaces/okrs-5eef636e504e9d001694ba22/roadmap">
+Each Carbon Design System release includes the releases of the libraries that
+are primarily maintained by the Carbon team. Those libraries are independently
+versioned following the industry-standard
+[semantic versioning](https://semver.org) specification.
 
-![ZenHub icon](images/z-icon.svg)
-
-  </ResourceCard>
-</Column>
-<Column colMd={4} colLg={4} noGutterSm>
-  <ResourceCard
-    subTitle="Squad 2020 Q4 OKRs"
-    href="https://app.zenhub.com/workspaces/okrs-5eef636e504e9d001694ba22/board?labels=squad%3A%20system&milestones=2020Q4%232020-12-31&useDefaultFilterLogic=false&repos=273306655&showPRs=false&showEstimates=false&showReleases=false&showDependencies=false">
+Where the Carbon team strives for frictionless major version updates through
```

Comment:
Could we place this under a heading? It could be useful to be able to bookmark/link directly to this text in our support channels - slack, GH issues/discussions, etc.

```suggestion
### Carbon 11

Where the Carbon team strives for frictionless major version updates through
```